src/prefect/orion/database/migrations/versions/postgresql/2022_07_29_181713_fa985d474982_add_index_to_flow_run_infrastructure_.py

```diff
@@ -0,0 +1,33 @@
+"""add-index-to-flow-run-infrastructure_document_id
+
+Revision ID: fa985d474982
+Revises: add97ce1937d
+Create Date: 2022-07-29 18:17:13.174765
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "fa985d474982"
+down_revision = "add97ce1937d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(
```

Comment:
For Postgres I think this should be a concurrent index creation. It’s probable that no one using OSS has enough rows that this would take any time at all but for example in Cloud2 this would lock the Orion tables

Reply:
good call I added concurrent creation here